cordial_behavior/scripts/default_name/cordial_face_expression.json

```diff
@@ -0,0 +1,333 @@
+{"surprised_face":
```

Comment:
The format for this file might slightly change, which is what I was referencing when talking about the reading in of this file